src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocNodeImpl.java

```diff
@@ -72,6 +72,20 @@
      */
     private DetailNode parent;
 
+    /**
+     * Reflects if any of the successors of this {@code DetailNode} has property
+     * {@link #unclosedTag} set true.
+     */
+    private Boolean hasUnclosedTags;
```

Comment:
Even if follow this rule it is unclear why field name says whether tag**s** are present, and method name says only about one tag. Let's be consistent.